examples/ROOT_files/ROOT_STIR_consistency/DebugScripts/debug_consistency_with_root.py

```diff
@@ -110,6 +110,67 @@ def PointCloud3D(DataHandler):
     ax.set_ylabel('y (mm)')
     ax.set_zlabel('z (mm)')
     ax.legend(['Origin and Tolerance', 'Mean coords and stddev ', 'Voxel Positions'])
+    
+    plt.show()
+
+def PointCloud3D_all(point_sources_data):
+    # import matplotlib.pyplot as plt
+    # import random
+
+    fig = plt.figure(figsize=(10, 10))
+    ax = fig.add_subplot(projection='3d')
+    
+    for key in point_sources_data.keys():
+        DataHandler = point_sources_data[key]
```

Comment:
objects/variables should be `snake_case`, `PascalCase` is generally reserved for classes.

Reply:
Hi Robbie, thanks for your comments! I changed the names in the original function (def PointCloud3D(DataHandler)) accordingly, I hope it's more consistent with naming conventions now.